docs/site/angular-auth-oidc-client/docs/documentation/02-configuration.md

```diff
@@ -467,3 +467,12 @@ Allows multiple usage of refresh token. Refresh tokens which can be stored safel
 Activate this property only if your OIDC provider cannot be configured to rotate refresh tokens.
 
 Default = _false_
+
+### `disableIdTokenValidation`
+
+- Type: `boolean`
+- Required: `false`
+
+Disable validation for id_token. This is not recommended! You should always validate the id_token if returned.
```

Comment:
This does NOT disable the id token validation completely. The id token is still checked once in the beginning and after a refresh of the page This has to be mentioned here. What happens when the id token is expired after a refresh? Mention here, too.